integration_tests/README.md

```diff
@@ -0,0 +1,56 @@
+# Integration Tests
+
+## Running Tests
+
+To run integration tests:
+
+`make integration`
+
+## Making Tests
+
+### 1. Fixtures
+
+Fixtures Directory: `integration_tests/fixtures`
+
+- Folders in the fixtures directory will be built as docker images in alphabetical order
+- Each folder should contain a `Dockerfile` and any resources necessary to build the image
+- The resulting image will be named `cbfix_fixture_name`
+
+*Note*: Since fixtures are created in alpha order, they can have FROM directives for previously created images
+
+### 2. Tests
+
+Tests Directory: `integration_tests/tests`
+
+- Folders in the tests directory will be run as tests
```

Comment:
This also implies (as the TravisCI config file says) that we need golang on the test-running host. We should make sure to document those assumptions as well. (Or do we want to run the tests themselves in a container?)